redash/utils.py

```diff
@@ -95,9 +95,9 @@ def default(self, o):
         if isinstance(o, decimal.Decimal):
             return float(o)
 
-        if isinstance(o, datetime.date):
+        if isinstance(o, datetime.date) or isinstance(o, datetime.datetime):
```

Comment:
Yet without this line the deserializer yells it cant deserialize
datetime.datetime.

That's why I added it.

On Tue, Jun 16, 2015, 21:40 Arik Fraimovich <notifications@github.com>
wrote:

> In redash/utils.py
> <https://github.com/EverythingMe/redash/pull/455#discussion_r32555487>:
>
> > @@ -95,9 +95,9 @@ def default(self, o):
> >          if isinstance(o, decimal.Decimal):
> >              return float(o)
> >
> > -        if isinstance(o, datetime.date):
> > +        if isinstance(o, datetime.date) or isinstance(o, datetime.datetime):
>
> From what I understand datetime.datetime inherits datetime.date, and
> hence why isinstance(datetime.datetime.now(), datetime.date) == True. You
> see different behavior?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/EverythingMe/redash/pull/455/files#r32555487>.
>